documentation/how-to/search.en.md

```diff
@@ -41,6 +41,14 @@ actions :
     feature will be highlighted in yellow).
 2.  Tap on the *attributes button* to open its attributes.
 
+
+In addition, the Code Reader offers the capability to read NFC text tags.
```

Comment:
I would add a more prominent section about this. Remind the use they need to enable their NFC and that it is consuming more power.

Reply:
Great!